### PR TITLE
Add Spacelift Integration

### PR DESCRIPTION
**Pull request by theseanodell**
Signed-off-by: Sean O'Dell <seano@spacelift.io>

### CODE REVIEW

File: docs/website/data/integrations.yaml [hunk 1]
```diff
@@ -1417,6 +1417,30 @@ integrations:
...
+     description: Spacelift is a sophisticated CI/CD platform for Infrastructure as Code including Terraform, Pulumi, CloudFormation, Kubernetes, and Ansible. Spacelift utilizes Open Policy Agent to support a variety of policy types within the platform and Policy as Code for secure and compliance Infrastructure as Code.
+     labels:
+       category: Infrastructure as Code
+       layer: cicd
+     software:
+       - terraform
+       - pulumi
+       - cloudformation
+       - kubernetes
+       - ansible
```
Comment (ashutosh-narkar) [inline docs/website/data/integrations.yaml:1431]: You'll need to add an entry for `ansible` in the `software` section below.

**Comment by srenatus** [comment]
Any chance you find a minute to address this? Or maybe @cube2222 could have a look? 😃

**Comment by theseanodell** [comment]
@srenatus sorry for the delay. I disabled notifications. My apologies.

File: docs/website/data/integrations.yaml [hunk 2]
```diff
@@ -1822,3 +1846,9 @@ software:
   graphql:
     name: graphql
     link: https://graphql.org/
+  spacelift:
+    name: Spacelift
+    link: https://spacelift.io
+ ansible
```
Comment (srenatus) [inline docs/website/data/integrations.yaml:1852]: ⚡ missing `:` at the end here

Comment (srenatus) [reply docs/website/data/integrations.yaml:1852]: 😅 It took me a while to figure out what was wrong here...

**Comment by srenatus** [comment]
Please have another look at that integrations.yaml and its missing `:`.

**Comment by srenatus** [comment]
🧹 Feel free to re-open when you find the time to get back to this 😃

**Pull request closed without merging**